src/Chapter3.hs

```diff
@@ -910,8 +986,21 @@ Implement instances of "Append" for the following types:
 class Append a where
     append :: a -> a -> a
 
+newtype Gold = Gold Int
 
-{-
+instance Append Gold where
+  append (Gold g) (Gold h) = Gold $ g+h
+
+instance Append [a] where
+  append l1 l2 = l1 ++ l2
```

Comment:
You can eta-reduce the implementation of `append` for brevity. So you can see how easy it is to create instances :slightly_smiling_face: 
```suggestion
  append = (++)
```